src/web_resources/HangarXPLOR.DrawUI.js

```diff
@@ -1,14 +1,60 @@
 
 var HangarXPLOR = HangarXPLOR || {};
 
+function OverrideMarginsAndPaddings() {
+  var temp;
+
+  temp = $('#contentbody');
+  temp.css('padding-bottom', '15px');
+
+  temp = $('#contentbody > div');
+  temp.css('padding-left', '15px');
+  temp.css('padding-right', '15px');
+  temp.css('max-width', 'none');
+
+  temp = $('#billing > div.content.clearfix > div.sidenav');
+  temp.css('width', '330px');
+
+  temp = $('#billing > div.content.clearfix > div.inner-content');
+  temp.css('padding-left', '330px');
+
+  temp = $('#billing > div.content.clearfix > div.sidenav > ul > li.active > a > span.bg');
+  temp.css('width', '275px');
+
+  temp = $('#billing');
+  temp.css('padding', '15px');
+  temp.css('padding-top', '30px');
+
+  temp = $('#billing > div.content.clearfix');
+  temp.css('padding-top', '15px');
+  temp.css('padding-bottom', '15px');
+
+  temp = $('#billing .inner-content .top > .separator');
+  temp.css('margin-top', '10px');
+  temp.css('margin-bottom', '10px');
+
+  temp = $('#billing .content h3');
+  temp.css('margin-top', '10px');
+  temp.css('margin-bottom', '10px');
+}
+
```

Comment:
Oooh - I'll have to peek at these improvements...

Reply:
It might be undesirable, but it makes much better use of my 21:9 monitor